app/pages/BeelieversAuction/nft.tsx

```diff
@@ -92,75 +93,77 @@ export function NftDisplay({ nftId }: NftDisplayProps) {
+											: "bg-gradient-to-r from-primary/20 to-blue-400/20 text-primary border border-primary/30 shadow-md"
+									}`}
+								>
+									{nftType === "Mythic" ? "✨ " : "🐝 "}
+									{nftType}
+									{mythicName ? ": " + mythicName : ""}
+								</span>
+							</div>
+						)}
+						<div className="flex items-center justify-center gap-2 text-sm text-muted-foreground">
+							<span className="font-medium">Object ID: {trimAddress(nftId)}</span>
+						</div>
+					</div>
+					<div className="space-y-3">
+						{background && (
+							<div className="flex items-center justify-center gap-2 text-sm">
+								<span className="text-muted-foreground">Background:</span>
+								<span className="font-medium">{background}</span>
+							</div>
+						)}
 
-							{metadata.badges && metadata.badges.length > 0 && (
-								<div className="flex flex-wrap gap-1">
+						{metadata.badges && metadata.badges.length > 0 && (
+							<div className="space-y-2">
+								<span className="text-sm font-medium text-muted-foreground">Badges:</span>
+								<div className="flex flex-wrap justify-center gap-2 mt-2">
 									{metadata.badges.map((badge, index) => (
 										<span
 											key={index}
-											className="text-xs px-2 py-1 bg-blue-500/20 text-blue-400 rounded-full border border-blue-400/30"
+											className="inline-flex items-center gap-1 text-sm px-2 py-1 bg-gradient-to-r from-blue-500/20 to-purple-500/20 text-blue-400 rounded-full border border-blue-400/30 font-medium shadow-md hover:scale-105 transition-transform duration-200"
 										>
 											{badge}
 										</span>
 									))}
 								</div>
-							)}
-
-							<div className="text-xs text-muted-foreground">
-								Beeliever #{metadata.token_id} <br />
-								Object ID: {nftId}
 							</div>
-						</div>
+						)}
+					</div>
 
-						{/* Move the View NFT button to bottom and fix styling */}
-						<div className="pt-2">
-							<a
-								href={mkSuiVisionUrl(nftId, network)}
-								target="_blank"
-								rel="noopener noreferrer"
-								className="inline-flex items-center gap-2 px-4 py-2 bg-primary/90 text-white border border-white/10 rounded-lg hover:bg-primary transition-colors font-medium text-sm shadow-[inset_0_4px_10px_0_rgba(255,255,255,0.25),inset_0_-4px_10px_0_rgba(255,255,255,0.15)]"
-							>
-								<ExternalLink size={16} />
-								View on SuiVision
-							</a>
-						</div>
+					<div className="pt-2 flex justify-center">
+						<a
+							href={mkSuiVisionUrl(nftId, network)}
+							target="_blank"
+							rel="noopener noreferrer"
+							className="inline-flex items-center gap-3 px-6 py-3 bg-gradient-to-r from-primary to-primary/80 text-white rounded-xl hover:from-primary/90 hover:to-primary/70 transition-all duration-300 font-semibold text-sm shadow-lg hover:shadow-xl hover:scale-105 border border-white/10"
```

Comment:
use button or a normal link. We have classes for that.

Reply:
Used button here